tests/kernel/arm_runtime_nmi/src/arm_runtime_nmi.c

```diff
@@ -15,6 +15,11 @@
 #include <ztest.h>
 #include <tc_util.h>
 
+/* on v8m arch the nmi pend bit is renamed to pend nmi map it to old name */
```

Comment:
This would work. Check, however, my proposal for a minor implementation and API change